layouts/partials/hooks/head-end.html

```diff
@@ -6,3 +6,8 @@
 <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" />
 {{ end }}
 
+
+{{- template "_internal/opengraph.html" . -}}
+{{- template "_internal/google_news.html" . -}}
+{{- template "_internal/schema.html" . -}}
+{{- template "_internal/twitter_cards.html" . -}}
```

Comment:
Is this required to be added? Given that it's in https://github.com/google/docsy/blob/master/layouts/partials/head.html#L19-L22 we should be good, right?